packages/@react-stately/virtualizer/src/Virtualizer.ts

```diff
@@ -321,6 +321,9 @@ export class Virtualizer<T extends object, V, W> {
   }
 
   private _renderView(reusableView: ReusableView<T, V>) {
+    if (!reusableView.layoutInfo) {
+      return;
+    }
```

Comment:
We were running into issues where layoutInfo wasn't available and it would cause errors. It makes sense that if layoutInfo isn't available then we should just return, but this is one possible section where we're avoiding some kind of deeper underlaying problem and if you have any ideas on why this might be happening and how we should fix it instead then we'd be happy to work together through this.

Reply:
I believe the "loads more on scroll when contentSize.height < rect.height * 2" storybook test is one that fails if you remove these changes - if someone wanted to test the behavior.